Dockerfile-dev

```diff
@@ -0,0 +1,21 @@
+FROM node:16-alpine
+
+RUN apk update
+RUN apk add bash
+
+RUN npm i -g pnpm
+
+WORKDIR app
+
+COPY .npmrc package.json pnpm-lock.yaml ./
+RUN pnpm i -P --frozen-lockfile --shamefully-hoist --reporter=silent
+
+COPY config.dev.json webpack.server.config.js ./
+COPY static static
+COPY lib lib
+COPY build build
```

Comment:
This assumes the `build` script is run beforehand.
This build step doesn't really belong to a `Dockerfile` imo.
Better to keep such jobs outside to keep the images small (no dev deps) and simple.